src/AddTodoForm.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState('');
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    onAddTodo({ title: todoTitle, id: Date.now() });
+    setTodoTitle(''); 
+  };
+
+  return (
+    <form onSubmit={handleAddTodo}>
+      <input
```

Comment:
I mentioned the missing label in the previous submission so I won't hold you up on this one. Just make sure that it's incorporated when you merge previous lesson into this.